controller_manager:
  ros__parameters:
    update_rate: 100  # Hz
    use_sim_time: false

    diffbot_controller:
      type: diff_drive_controller/DiffDriveController

diffbot_controller:
  ros__parameters:

    publish_rate: 30.0 # You can set this higher than the controller manager update rate, but it will be throttled
    base_frame_id: base_link

    left_wheel_names: ['left_wheel_joint']
    right_wheel_names: ['right_wheel_joint']
    wheel_separation: 0.16
    wheel_radius: 0.03

    use_stamped_vel: false